net/net.sh

```diff
@@ -702,7 +702,7 @@ stopNode() {
   declare pid=$!
   ln -sf "stop-validator-$ipAddress.log" "$netLogDir/stop-validator-$pid.log"
   if $block; then
-    wait $pid
+    wait $pid || true
```

Comment:
Can you help me understand this, please?

Reply:
Basically, stopping the node results in a [pkill](https://github.com/solana-labs/solana/blob/master/net/remote/cleanup.sh#L24) across any of the possible processes running on the machine. pkill has exit code 1 in the case we don't find a matching process. We `|| true` here so that we don't exit the script (we have `set -e` active). You can find similar logic when we stop the entire network [here](https://github.com/solana-labs/solana/blob/master/net/net.sh#L728).